docs/querying/sql-multivalue-string-functions.md

```diff
@@ -36,14 +36,15 @@ sidebar_label: "Multi-value string functions"
 
 Druid supports string dimensions containing multiple values.
 This page describes the operations you can perform on multi-value string dimensions using [Druid SQL](./sql.md).
-See [Multi-value dimensions](multi-value-dimensions.md) for more information.
+See [SQL multi-value strings](./sql-data-types.md#multi-value-strings) and native [Multi-value dimensions](multi-value-dimensions.md) for more information.
 
 All "array" references in the multi-value string function documentation can refer to multi-value string columns or
-`ARRAY` literals.
+`ARRAY` types. Multi-value strings can also be converted to `ARRAY` types using `MV_TO_ARRAY`, see
+[Multi-value string functions](sql-multivalue-string-functions.md), [`ARRAY` data type documentation](./sql-data-types.md#arrays),
+and [array functions](./sql-array-functions.md) for additional details.
```

Comment:
Broke apart the run on sentence and moved the preposition to the front for clarity
```suggestion
`ARRAY` types. Multi-value strings can also be converted to `ARRAY` types using `MV_TO_ARRAY`.
For additional details, see [Multi-value string functions](sql-multivalue-string-functions.md),
[`ARRAY` data type documentation](./sql-data-types.md#arrays),
and [array functions](./sql-array-functions.md).
```